tools/record_example_video.py

```
@@ -0,0 +1,89 @@
+    ffmpeg = None
+    first_frame = blank = None
+    try:
+        for _ in range(args.num_frames):
+            if viewer.should_step():
+                example.step()
+            example.render()
+            frame = viewer.get_frame().numpy()  # (h, w, 3) uint8, top-left origin
+
+            if ffmpeg is None:
+                h, w = frame.shape[:2]
+                ffmpeg = subprocess.Popen(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)